_posts/2021-04-09-ugrc-ng911-update.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 'UGRC Next-Generation 911 Update'
+author:
+  display_name: Erik Neemann
+  email: eneemann@utah.gov
+date: 2021-04-09 08:30:12
+categories:
+  - Featured
+tags:
+  - ng911
+  - psap
+---
+
+Over the last several months the newly renamed Utah Geospatial Resource Center (UGRC) has been working hard to prepare the state for its transition to Next-Generation 911 (NG911).  With NG911, emergency calls will rely on geospatial data for address validation and call routing.  The old Enhanced 911 (E911) system, which relied on tabular data, will be gradually phased out after backing-up the geospatial data for a period of time.  This means that the dispatch center, known as a public safety answering point (PSAP), that receives your 911 call will be selected based on your current location and the PSAP's service boundary.  NG911 will also allow for dynamic changes to PSAP boundaries, allowing adjustments to call routing during large-scale emergencies, planned downtime, or periods of high call volume.
+
+To facilitate this paradigm shift, nearly everyone at UGRC has been working (directly or indirectly) to ensure NG911 is a success.  Dating back over the last few years, we've served as the geospatial subject matter expert for the Utah Communications Authority (UCA) as they reviewed contractor proposals.  More recently, Motorola was awarded as the service provider for network infrastructure, PSAP call-handling systems, and the geospatial backbone of NG911, known as the core services.  As the NG911 implementation date nears for Utah's first counties later this year, UGRC has been preparing the GIS data that will feed the system and working with 911 Datamaster to perform quality assurance/quality control (QA/QC).  Specifically, we've built the following [datasets](https://gis.utah.gov/data/911/) that will be included in the initial NG911 rollout:
```

Comment:
This link format should be updated since it is an internal link.

Reply:
Oh shoot. I was too worried about the markdown syntax. I'll fix this.